ui/src/api/mixins/Table.ts

```diff
@@ -0,0 +1,27 @@
+import { Resource } from 'alcaeus'
+import * as ns from '@cube-creator/core/namespace'
+import { schema } from '@tpluscode/rdf-ns-builders'
+import { Table, Source } from '@/types'
+import { findOperation } from '../common'
+
+export type Constructor = new (...args: any[]) => Resource;
+
+export default function Mixin<Base extends Constructor> (base: Base) {
```

Comment:
I see what you did there 🙂. The patterns goes like this

```suggestion
import { Constructor } from '@tpluscode/rdfine'

export default function Mixin<Base extends Constructor<Resource> (base: Base) {
```

Reply:
Alright, will change 👍 
I guess it was a pattern I ported from the first project.